packages/react-cosmos/src/components/proxies/PropsProxy.jsx

```diff
@@ -10,11 +10,15 @@ export default class PropsProxy extends React.Component {
       fixture,
       onComponentRef,
     } = this.props;
+    let children;
+    if (Array.isArray(fixture.children)) {
+      children = [].slice.call(fixture.children);
+    }
 
     return React.createElement(component, {
       ...fixture,
```

Comment:
I think the destructuring of children is optional: https://jsbin.com/yikulexefe/edit?js,output – which means the 3rd argument from `createElement` can be a list (or single element, or `undefined`).

With this in mind, do you see a problem with this?
```js
    React.createElement(component, {
      ...omit(fixture, 'children'),
      ref: onComponentRef,
    }, fixture.children);
```